lib/Validator.ts

```diff
@@ -185,22 +193,22 @@ export default class Validator {
       this.logger.error(error);
     }
 
-    this.prevPausedOrPlaying = 'none';
+    this.prevPlayPauseStatus = PlayPauseStatus.NONE;
 
     return output;
   }
 
   _invalidPausedPlayingEvent = (event: string): boolean => {
     if (event === 'paused') {
-      if (this.prevPausedOrPlaying === 'playing') {
-        this.prevPausedOrPlaying = 'paused';
+      if (this.prevPlayPauseStatus === PlayPauseStatus.PLAYING) {
+        this.prevPlayPauseStatus = PlayPauseStatus.PAUSED;
         return false;
       }
       return true;
     }
     if (event === 'playing') {
-      if (this.prevPausedOrPlaying === 'paused' || this.prevPausedOrPlaying === 'none') {
-        this.prevPausedOrPlaying = 'playing';
+      if (this.prevPlayPauseStatus === PlayPauseStatus.PAUSED || this.prevPlayPauseStatus === PlayPauseStatus.NONE) {

```

Comment:
When multiple comparisons against the same variable I personally prefer the pattern
`[PlayPauseStatus.PAUSED, PlayPauseStatus.NONE].includes(this.prevPlayPauseStatus)`
Though that might just be my opinion

Reply:
I like this